weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewer.tsx

```diff
@@ -137,11 +137,19 @@ export const ObjectViewer = ({
     }
     let resolved = data;
     let dirty = true;
+    const resolvedRefPaths = new Set<string>();
     const mapper = (context: TraverseContext) => {
-      if (isRef(context.value) && refValues[context.value] != null) {
+      const parentPathStr = context.path
+        .ancestor(context.path.length() - 1)
```

Comment:
Maybe add a `parent` method to `ObjectPath` that does this

Reply:
good idea, done